### ISSUE TITLE
Webhook Notification supports body configuration.

### ISSUE THREAD

**Issue by tianfeng98** [jinweijie/notify-me#4]
Hi @jinweijie , 

Thank you so much for creating this app; it’s a wonderful tool that can help my grandmother avoid scams. Previously, I used email to receive text messages and phone calls, but since email only allows personal message reception, there were times when I struggled to manage everything while busy. 

To make things easier, I set up a DingTalk group and wanted to forward messages through a group robot, but perhaps because the robot needs to configure the request body, the webhook push failed and the robot did not respond. 

I hope you can help me, and I express my sincere gratitude on behalf of myself and my family.

**Comment by jinweijie** [comment]
@tianfeng98 I am happy to help, could you send me the sample payload which needs to be sent to DingTalk? So that I can have some tests? Thanks.

**Comment by tianfeng98** [comment]
For my needs, sending a text message is enough. For example, refer to your test pyload：
```json
{
  "sender": "Tester",
  "message": "This is a test message",
  "type": "SMS"
}
```
I can send a DingTalk message like this:
```json
{
  "msgtype": "text",
  "text": {
     "content": "[SMS][Tester]This is a test message"
  }
}
```
For complete message configuration, please refer to the [official DingTalk documentation](https://open.dingtalk.com/document/orgapp/custom-robots-send-group-messages?spm=ding_open_doc.document.0.0.75237f7f5WBmsS).

For developers, I suggest that in addition to providing preset methods such as Bark, Email, Webhook, etc., you can also provide custom request configuration. This way, if others have needs like mine, they don't have to bother you.

I wanted to help you contribute code, but unfortunately I'm just a javascript developer, I can't write kotlin code.😞

**Comment by jinweijie** [comment]
@tianfeng98 ok, understood, in this case I'll add another method (http protocol) to forward the message.

**Comment by jinweijie** [comment]
Hi @tianfeng98 I've add the http protocol support and released a new version (1.3.0), please try, thanks.
